src/renderer/services/binance/balances.ts

```diff
@@ -1,11 +1,24 @@
-import { triggerStream } from '../../helpers/stateHelper'
+import { observableState } from '../../helpers/stateHelper'
 import * as C from '../clients'
+import { ReloadType } from '../types'
 import { client$ } from './common'
 
-// `TriggerStream` to reload `Balances`
-const { stream$: reloadBalances$, trigger: reloadBalances } = triggerStream()
+/**
+ * `ObservableState` to reload `Balances`
+ * Sometimes we need to have a way to understand if it simple "load" or "reload" action
+ * e.g. @see src/renderer/services/wallet/balances.ts:getChainBalance$
+ */
+const { get$: reloadBalances$, set: setReloadBalances } = observableState<ReloadType>('load')
```

Comment:
(here and further) we can not use `observableState<boolean>` as ` C.balances$` requires `reloadStream$` to be `TriggerStream` type which is actually `Observable<string>`